composer.json

```diff
@@ -19,8 +23,22 @@
   },
   "require": {
     "php": ">=5.4",
+    "behat/behat": "~3.1.0-rc2",
     "behat/mink": "^1.7",
```

Comment:
mink may not be required for this as a main dependency - only dev (for tests) -- see my comment below